docs/core/tools/dotnet-list-package.md

```diff
@@ -0,0 +1,102 @@
+
+```output
+Project 'SentimentAnalysis' has the following package references
+   [netcoreapp2.1]:
+   Top-level Package               Requested   Resolved
+   > Microsoft.ML                  0.9.0       0.9.0
+   > Microsoft.NETCore.App   (A)   [2.1.0, )   2.1.0
+
+(A) : Auto-referenced package.
+```
+
+The **Requested** column refers to the package version specified in the project file and can be a range. The **Resolved** column lists the version that the project is currently using and is always a single value. The packages displaying an `(A)` right next to their names represent [implicit package references](csproj.md#implicit-package-references) that are inferred from your project settings (`Sdk` type, `<TargetFramework>` or `<TargetFrameworks>` property, etc.)
+
+Use the `--outdated` option to find out if there are newer versions available of the packages you're using in your projects. By default, `--outdated` lists the latest stable packages unless the resolved version is also a prerelease version. To include prerelease versions when listing newer versions, also specify the `--include-prerelease` option.
+
+## Arguments
+
+`PROJECT | SOLUTION`
+
+The project or solution file to operate on. If not specified, the command searches the current directory for one. If more than one solution or project is found, an error is thrown.
+
+## Options
+
+* **`--config <SOURCE>`**
+
+  The NuGet sources to use when searching for newer packages. Requires the `--outdated` option.
+
+* **`--framework <FRAMEWORK | FRAMEWORK/RID>`**
+
+  Displays only the packages applicable for the specified [target framework](../../standard/frameworks.md) or [Runtime Identifier (RID)](../rid-catalog.md) to list the packages for. To specify multiple frameworks, repeat the option multiple times. For example: `--framework netcoreapp2.2 --framework netstandard2.0`.
+
+* **`-h|--help`**
+
+  Prints out a short help for the command.
+
+* **`--highest-minor`**
+
+  Considers only the packages with a matching major version number when searching for newer packages. Requires the `--outdated` option.
+
+* **`--highest-patch`**
+
+  Considers only the packages with a matching major and minor version numbers when searching for newer packages. Requires the `--outdated` option.
+
+* **`--include-prerelease`**
+
+  Considers packages with prerelease versions when searching for newer packages. Requires the `--outdated` option.
+
+* **`--include-transitive`**
+
+  Lists transitive packages, in addition to the top-level packages. When specifying this option, you get a list of packages that the top-level packages depend on.
+
+* **`--outdated`**
+
+  Lists packages that have newer versions available.
+
+* **`-s|--source <SOURCE>`**
+
+  The NuGet sources to use when searching for newer packages. Requires the `--outdated` option.
```

Comment:
There's a new switch added in 3.0.1xx 
https://github.com/dotnet/cli/pull/10994

`--interactive`

Allows the command to stop and wait for user input or action (for example to complete authentication)

Reply:
Will this be available with preview 4 @nkolev92? I can create an issue once that's out to add that flag.

Reply:
Yeah, it will be.